Jenkinsfile

```diff
@@ -0,0 +1,29 @@
+pipeline {
+    agent any
+    environment {
+      HOME = "${env.WORKSPACE}"
+      DOCKER_REGISTRY = 'docker.chameleoncloud.org'
+      DOCKER_REGISTRY_CREDS = credentials('kolla-docker-registry-creds')
+    }
+
+    stages {
+      stage('docker-setup') {
+        steps {
+          sh 'docker login --username=$DOCKER_REGISTRY_CREDS_USR --password=$DOCKER_REGISTRY_CREDS_PSW $DOCKER_REGISTRY'
+        }
+      }
+      stage('build-and-publish') {
+        steps {
+          sh 'docker build -t hammers .'
```

Comment:
If you make this `docker build -t $DOCKER_REGISTRY/hammers:latest`, the next `tag` line is unnecessary.